.redhat-ci.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# F25 currently has 2.2.1, so install from pypi
+pip install ansible==2.2.2.0
```

Comment:
Shouldn't we use `pip install -r requirements.txt` instead?
It's going to be one more place to maintain when bumping the ansible version as we're doing in https://github.com/openshift/openshift-ansible/pull/3986.